app/components/Welcome.js

```diff
@@ -15,6 +16,17 @@ export default class Welcome extends Component {
       return (<Spinner />)
     }
 
+    // show welcome page only if the user did not hit
+    // home page ever
+    let electronStore = electron.remote.getGlobal('sharedObj').electronStore
+    if (electronStore.has('not-first-time') && electronStore.get('not-first-time') === true) {
```

Comment:
Let the redirection be in home as showing welcome is the exception and make the initial redirection for home component